### PR TITLE
Use sx instead of makestyles

### PR DESCRIPTION
**Pull request by nicarl**
### Summary of changes

Replace usage of `makestyles` with `sx`

### Context and reason for change
MUI will deprecate `makestyles`.

### How can the changes be tested
Open the app and check the styling.

### CODE REVIEW

**Comment by nicarl** [comment]
The report view still has some issues.

**How it looks**
current_styling

**How it should look like**
how_it_should_look_like

File: src/Frontend/Components/AttributionColumn/LicenseSubPanel.tsx [hunk 1]
```diff
@@ -76,30 +75,33 @@ export function LicenseSubPanel(props: LicenseSubPanelProps): ReactElement {
...
+          ...licenseSubPanelClasses.expansionPanel,
+        }}
         elevation={0}
         key={'License'}
         expanded={props.isLicenseTextShown}
         onChange={doNothing}
       >
         <MuiAccordionSummary
-          classes={{ root: classes.expansionPanelSummary }}
+          sx={{ root: licenseSubPanelClasses.expansionPanelSummary }}
```
Comment (benedikt-richter) [inline src/Frontend/Components/AttributionColumn/LicenseSubPanel.tsx:91]: this probably won't work

File: src/Frontend/Components/AttributionColumn/LicenseSubPanel.tsx [hunk 2]
```diff
@@ -118,10 +120,15 @@ export function LicenseSubPanel(props: LicenseSubPanelProps): ReactElement {
             }
           />
         </MuiAccordionSummary>
-        <MuiAccordionDetails classes={{ root: classes.expansionPanelDetails }}>
+        <MuiAccordionDetails
+          sx={{ root: licenseSubPanelClasses.expansionPanelDetails }}
```
Comment (benedikt-richter) [inline src/Frontend/Components/AttributionColumn/LicenseSubPanel.tsx:124]: this probably won't work

File: src/Frontend/Components/AttributionColumn/shared-attribution-column-styles.ts [hunk 3]
```diff
@@ -3,30 +3,29 @@
...
+    paddingBottom: '0px',
     background: OpossumColors.lightestBlue,
     border: `1px ${OpossumColors.lightestBlue} solid`,
   },
   displayRow: {
     display: 'flex',
   },
   textBox: {
-    marginBottom: 10,
+    marginBottom: '4px',
```
Comment (benedikt-richter) [inline src/Frontend/Components/AttributionColumn/shared-attribution-column-styles.ts:20]: why is this modified?

**Comment by benedikt-richter** [comment]
> The report view still has some issues.
> 
> **How it looks** current_styling
> 
> **How it should look like** how_it_should_look_like


The styling is now fixed.
Also the tests that were broken are fixed.

**Review by leslielazzarino — APPROVED** [review]
